src/routes/reference/pca.svx

```diff
@@ -12,4 +12,45 @@ flair: reference
+
+As this is often how PCA is used, FluidPCA allows the user to specify the number of output dimensions desired, so that `transform` operates as a dimensionality reduction algorithm (other dimensionality reduction algorithms in FluCoMa include [UMAP](/reference/umap) and [MDS](/reference/mds). After FluidPCA transforms a DataSet it also returns the _explained variance_: the fraction (between 0 and 1) of the total variance in the DataSet accounted for using the reduced number of dimensions it has transformed the DataSet into. Values close to 1 mean that most of the variance in the original data is represented while number near 0 mean most of the variance in the original data is not represented in the transformed DataSet. When using PCA, knowing the explained variance is quite important. Reducing the number of dimensions while loosing most of the variance will likely be problematic further in the processes. Being able to reduce the number of dimensions while retaining most of the variance can help avoid the curse of dimensionality, speed up future computations, and/or remove redundancy or noise in the data. Aiming to retain an explained variance of 0.99, 0.95, or 0.9 are common.
+
+## `whiten`
+
+PCA "whitening" can be turned on by setting the `whiten` parameter to 1. "Whitening" ensures that not only will the principal components be uncorrelated, but also they will all have unit variance (they'll all cover the same range of 1). "Whitening" is sometimes also referred to as `sphering` meaning that it's making the data "spherical" in the sense that each principal component (each new dimension) has the same spread, just as a sphere has the same width in each of its three dimensions.
+
+Whitening looses some information during the transformation process (essentially, the relative scales of the principal components), however, whitening may improve the accuracy of further computations such as classification. If there is noise in the data, whitening might amplify this noise by scaling it to be the same size as other other information, making it as prominent as the relevant information, therefore whitening will not _always_ improve data.
+
+## Related Resources
+
+<ResourceLink
+title='Principal Component Analysis (PCA) - Computerphile'
+url='https://www.youtube.com/watch?v=TJdH6rPA-TI'
+blurb='YouTube video containing great visual explanations of PCA'
+/>
+
+<ResourceLink
+title='PCA Whitening'
+url='http://ufldl.stanford.edu/tutorial/unsupervised/PCAWhitening/'
+blurb='Some of the maths behind PCA Whitening.'
```

Comment:
is Whitening here a proper noun?

Reply:
I don't understand what you're asking? Are you suggesting that whitening should be lowercase?

Reply:
If it is not a proper noun then yeah